README.md

```diff
@@ -4,13 +4,9 @@
 
 **Temporian** is a Python package for **feature engineering of temporal data**, focusing on providing a **simple and powerful API**, a first-class **iterative development** experience, **efficient and well-tested implementations** of common and not-so-common temporal data preprocessing functions, and **preventing common modeling errors**.
 
-## Why Temporian?
-
-Temporian helps you **focus on high-level modeling**.
-
-Temporal data processing is commonly done with generic data processing tools. However, this approach is often tedious, error-prone, and requires engineers to learn and re-implement existing methods. Additionally, the complexity of these tools can lead engineers to create less effective pipelines in order to reduce complexity. This can increase the cost of developing and maintaining performant ML pipelines.
+## Documentation
```

Comment:
would definitely move after the installation part, and I think after the getting started too